docs/docs/development/setup.mdx

```diff
@@ -203,21 +169,7 @@ Zephyr™ application is in the `app/` source directory:
 ### Step into the repository
 
 <OsTabs>
-<TabItem value="debian">
-
-```sh
-cd zmk
-```
-
-</TabItem>
-<TabItem value="raspberryos">
-
-```sh
-cd zmk
-```
-
-</TabItem>
```

Comment:
I suppose this was deleted by accident, currently it renders empty under the tab.

Reply:
This and the below next comment were resolved in reshuffling.